target/linux/rockchip/image/armv8.mk

```diff
@@ -115,6 +115,16 @@ define Device/radxa_e25
 endef
 TARGET_DEVICES += radxa_e25
 
+define Device/radxa_rock-3a
+  DEVICE_VENDOR := Radxa
+  DEVICE_MODEL := ROCK 3A
+  SOC := rk3568
+  DEVICE_DTS := rockchip/rk3568-rock-3a
+  UBOOT_DEVICE_NAME := rock-3a-rk3568
```

Comment:
```suggestion
```

Can be omitted as they're default values.

https://github.com/openwrt/openwrt/blob/1664e506acb51bdce1cdef4515efa38377d9b60a/target/linux/rockchip/image/Makefile#L53-L54

Reply:
thanks, fixed